src/videojs-http-streaming.js

```diff
@@ -511,6 +511,12 @@ class VhsHandler extends Component {
       videojs.log.warn('Using hls options is deprecated. Use vhs instead.');
     }
 
+    // if a tech level `initialBandwidth` option was passed
```

Comment:
Probably worth documentating and mentioning that it takes precedence over VHS level `bandwidth` but not source level `bandwidth`.